<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_Refund</class>
 <widget class="QDialog" name="Admin_Refund">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

position: relative;
width: 1280px;
height: 720px;

background: #FFFFFF;</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>199</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 1280px;
height: 199px;
left: 0px;
top: 0px;
/* Refund */


position: absolute;
width: 360px;
height: 107px;
left: calc(50% - 360px/2);
top: 46px;

font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 96px;
line-height: 116px;
display: flex;
align-items: center;
text-align: center;

color: #FFFFFF;

background: #252748;</string>
   </property>
   <property name="text">
    <string>REFUND</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>202</y>
     <width>144</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;

width: 144px;
height: 54px;
left: 53px;
top: 202px;
border:none;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 32px;
line-height: 39px;
display: flex;
align-items: center;
text-align: center;

color: #000000;</string>
   </property>
   <property name="text">
    <string>Reports</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>1280</width>
     <height>521</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>331</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
position: absolute;
width: 637px;
height: 428px;
left: 53px;
top: 256px;
font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 32px;
background: #D9D9D9;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Inter'; font-size:32px; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:42px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>310</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 637px;
height: 428px;
left: 53px;
top: 256px;

background: #D9D9D9;position: absolute;
width: 320px;
height: 51px;
left: 747px;
top: 277px;

font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 42px;
line-height: 51px;
/* identical to box height */

display: flex;
align-items: center;
text-align: center;

color: #000000;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>PLAYER NAME</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>400</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 234px;
height: 87px;
left: 892px;
top: 369px;

background: #252748;
position: absolute;
width: 637px;
height: 428px;
left: 53px;
top: 256px;

position: absolute;
width: 320px;
height: 51px;
left: 747px;
top: 277px;

font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 42px;
line-height: 51px;
/* identical to box height */

display: flex;
align-items: center;
text-align: center;
colo:#FFF;

</string>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>400</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 234px;
height: 87px;
left: 892px;
top: 369px;

background: #252748;
position: absolute;
width: 637px;
height: 428px;
left: 53px;
top: 256px;

position: absolute;
width: 320px;
height: 51px;
left: 747px;
top: 277px;

font-family: 'Inter';
font-style: normal;
font-weight: 900;
font-size: 42px;
line-height: 51px;
/* identical to box height */

display: flex;
align-items: center;
text-align: center;


</string>
   </property>
   <property name="text">
    <string>Decline</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>textEdit</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
